Cargo.toml

```diff
@@ -17,7 +17,7 @@ edition = "2021"
 beerus-core = { path = "./crates/beerus-core" }
 
 # Helios revision is supposed to be changed by the corresponding tag once the revision is released. Next expected tag: 0.5.0.
-helios = { git = "https://github.com/a16z/helios", rev = "f63e4142c8c8094cdf94cf8d1ef1cb637a7eac98" }
+helios = { git = "https://github.com/glihm/helios", rev = "688ed805a142ef6ead361889cc6d42a4cdede918" }
```

Comment:
I see that you are pointing directly at a version in your personal github profile, what is the plan with that and porting back to main helios repo?

Reply:
The plan is that to have wasm32 PR accepted i must check with the maintainer how to fix an issue related to threading in wasm32. This may take a long time so it means if we wait this pr to be merged... we may have during a long time a CI that will fail.

The idea is that I'll keep my repo updated with Helios main, and we will get back to Helios main when the wasm32 proble is fixed.